src/sass/components/_animations.scss

```diff
@@ -0,0 +1,111 @@
+%anim {
+    // transition: all 0.25s linear;
+    transition: background-color 0.25s linear, color linear 0.25s, border-color linear 0.25s;
+}
+
+.btn-outline {
+    @extend %anim;
+}
+
+.btn-main {
+    @extend %anim;
+}
+
+
+header {
+    .top-bar {
+        ul {
+            li {
+                a {

```

Comment:
łatwiej chyba będzie dodać klasę na to a i ostylować po tej klasie, bo robi nam się tu niezła choinka i poniżej wszystkie inne też spróbuj po mniejszej ilości zagnieżdżeń - max zagnieżdżeń to pewnie w okolicach 3 wcięć, w metodologiach CSSów takich jak BEM czy SMACSS jedna, dwie klasy wystarczą żeby ostylować dany element, spróbujmy też iść w tą stronę